tests/integrated/test-stopCheck-file/runtest

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env python3
 
-#requires: not travis
+# requires: not travis
+# h5py doesn't load the data as expected
```

Comment:
Might be worth checking this is still the case

Reply:
This is with python3-h5py-2.10.0
If the data-length is 1, it is returned as a number, rather, then an array of length 1.

Reply:
Fixed. It couldn't tell the difference between a time-dependent scalar with only one value, and an time-independent scalar